.github/workflows/ci.yml

```diff
@@ -0,0 +1,21 @@
+name: CI
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+jobs:
+  codeql:
+    uses: philips-software/roslyn-analyzers/.github/workflows/codeql-analysis.yml@bcollamore-ci
+
+  pinnedactions:
+    if: ${{ github.event_name == 'pull_request' }}
+    uses: philips-software/roslyn-analyzers/.github/workflows/pinned-actions.yml@bcollamore-ci
+
+  build:
+    uses: philips-software/roslyn-analyzers/.github/workflows/dotnetcore.yml@bcollamore-ci
+
+  sonarcloud:
+    uses: philips-software/roslyn-analyzers/.github/workflows/sonarcloud.yml@bcollamore-ci
```

Comment:
So, you will remove the `@bcollamore-ci` after merging to develop right?

Reply:
That's correct. (1) Remove "Required" branch protection from 'old' workflows.  (Will need another Approval.) (2) Update this file to point to master (which will fail CI in this PR because the workflows aren't reusable on master yet. (3) Clean up / re-create "Required" workflows branch protection on the new ci.yml workflow.